README.rst

```diff
@@ -52,6 +60,19 @@ Examples_ that demonstrate how to use it.
 .. _Examples: https://skypy.readthedocs.io/en/stable/examples/index.html
 
 
+Get in Touch
+------------
+
+You are welcome to get in touch about the SkyPy package and code
+using our `Discussions page`_.
+
+Or for any other questions about the project in general, please get in touch with
+the current SkyPy co-ordinators:
+`coordinators@skypyroject.org`_.
+
+ .. _Discussions page: https://github.com/skypyproject/skypy/discussions 
+ .. _coordinators@skypyroject.org: mailto:coordinators@skypyproject.org
```

Comment:
Do these links function?

Reply:
Not yet. It would be actions on (I think) @ntessore and @aamara to make them work.

Reply:
it needs to be done by the admin of the repository, and I don't know who that is either

Reply:
Discussions tab works, thanks! @aamara can the email be set up?

Reply:
If the domain is skypyproject.org then I can set it up. But Adam wants to start a poll about which domain people want to use first.

Reply:
We could set it up with that domain now and then have a 'change all links to the correct domain' PR later if necessary?